docs/index.md

```diff
@@ -15,6 +15,11 @@ SPDX-License-Identifier: MPL-2.0
 Power Grid Model IO is a tool to convert grid data to and from the native data format of [power-grid-model](https://github.com/PowerGridModel/power-grid-model).
 Currently, conversions from Vision excel exports is possible. Pandapower conversions are under development.
 
+## Citations
+
+If you are using Power Grid Model IO in your research work, please consider citing our library using the references in [Citation](citations/CITATION.md)
```

Comment:
I think in repo PGM you actually download citation.md from .github, and then refer it in the index. Why do you do different here?

Reply:
whoops forgot to push